cypress/e2e/warehouse.spec.cy.js

```diff
@@ -29,32 +30,27 @@ describe("Warehouse section", () => {
         .should("be.visible")
         .click();
 
-    cy.get('li[role="option"]')
-        .should("be.visible")
+    clickCheck({ clickElem: 'li[role="option"]', contains: "Warehouse Activity" });
+
+    // Check that text "Warehouse Activity" is visible
+    cy.get("span")
         .contains("Warehouse Activity")
-        .should("be.visible")
-        .click();
+        .should("be.visible");
```

Comment:
there seems to be a click that was taken out here

Reply:
yea, the diff shows it in a weird way. There was a click that I replaced with a call to clickCheck() and then I follow up with checking if we can actually see "Warehouse Activity" (that might be redundant actually, but not harmful ... I think)